scripts/ci-install-macos-latest.sh

```diff
@@ -3,3 +3,5 @@ sudo ln -s $(pwd)/nvim-osx64/bin/nvim /usr/local/bin
 mkdir -p ~/.local/share/nvim/site/pack/nvim-treesitter/start
 ln -s $(pwd) ~/.local/share/nvim/site/pack/nvim-treesitter/start
 
+brew update
+brew install tree-sitter-cli@${TREE_SITTER_CLI_TAG}
```

Comment:
That doesn't work -- homebrew does not have multiple versions except for some special formulas that can be installed in parallel (like Python). You can only install the latest version (without `@${TAG}`).

Reply:
Also, the formula is just called `tree-sitter`.

Reply:
And you can skip `brew update`, that is implied by `brew install`.

Reply:
there were some challenges with node on MacOS/Windows. At the moment, generate from grammar is only done on Ubuntu. When node works you can also do the same `npm` installation like Ubuntu.

Reply:
`brew install node` should do the trick (that's what I use locally).